src/llama_recipes/finetuning.py

```diff
@@ -163,7 +149,7 @@ def main(**kwargs):
             wandb_run.config.update(peft_config)
         model.print_trainable_parameters()
 
-    hsdp_device_mesh_plan = None
+    hsdp_device_mesh = None
```

Comment:
This reintroduces an issue #402 where we set hsdp_device_mesh to None and then call it in line 154.
```suggestion
    hsdp_device_mesh_plan = None
```